bin/hsw-cli

```diff
@@ -592,93 +592,76 @@ class CLI {
       case 'rescan':
         await this.rescan();
         break;
+      case 'wallets':
+        await this.getWallets();
+        break;
+      case 'mkwallet':
+        await this.createWallet();
+        break;
+      case 'resend':
+        await this.resend();
+        break;
```

Comment:
Ok I didn't realize before but we have duplicate `resend` commands, with two functions in `hsw-cli` `resend()` and `resendWallet(id)`

The admin command resends from all wallets and is documented: https://hsd-dev.org/api-docs/#wallet-resend

in hsd: `this.post('/resend' ...`

The non-admin command requires a wallet id (or defaults to `primary`), and it is not documented.

in hsd: `this.post('/wallet/:id/resend ...`

So I think because of the admin shift/unshift thing the per-wallet resend was just never used (or not even accessible?) SO GOOD NEWS --> you get to make up a name for the command that just executes the resend function on one specific wallet :-)

Reply:
Okay, the per-wallet command seems to be not so useful, as resend all is innocent in my opinion (if tx was already tried to broadcast, then the user was indeed willing to do it, so there is no harm doing it again). So it the command can be a bit cryptic comparing to just `resend` for all wallets.

Following the `mkwallet` I suggest `resendwallet`.

Reply:
By the way, providing unexsiting id will not cause any error. 
As there is no incorrect id handling for wallet actions (https://github.com/handshake-org/hsd/blob/master/lib/wallet/http.js#L801)